packages/workflow/src/Interfaces.ts

```diff
@@ -39,14 +39,15 @@ export type IAllExecuteFunctions =
 
 export type BinaryFileType = 'text' | 'json' | 'image' | 'audio' | 'video' | 'pdf' | 'html';
 export interface IBinaryData {
-	[key: string]: string | undefined;
+	[key: string]: string | number | undefined;
 	data: string;
 	mimeType: string;
 	fileType?: BinaryFileType;
 	fileName?: string;
 	directory?: string;
 	fileExtension?: string;
 	fileSize?: string; // TODO: change this to number and store the actual value
+	fileSizeRaw?: number;
```

Comment:
This entire change could be a separate PR, since it's not specific to the Slack changes.